src/main/java/uk/gov/pay/api/model/CreatePaymentRequest.java

```diff
@@ -29,12 +38,23 @@
     public static final String PREFILLED_ADDRESS_CITY_FIELD_NAME = "city";
     public static final String PREFILLED_ADDRESS_POSTCODE_FIELD_NAME = "postcode";
     public static final String PREFILLED_ADDRESS_COUNTRY_FIELD_NAME = "country";
+    
+    @Min(value = AMOUNT_MIN_VALUE, message = "Must be greater than or equal to {value}")
```

Comment:
Shouldn't `AMOUNT_MIN_VALUE` be defined in this class (CreatePaymentRequest) rather than PaymentRequestValidator? My understanding is that PaymentRequestValidator will be deprecated.

Reply:
Same for AMOUNT_MAX_VALUE, REFERENCE_MAX_LENGTH, DESCRIPTION_MAX_LENGTH and AGREEMENT_ID_MAX_LENGTH.